unit_tests/test_base_version.py

```diff
@@ -33,7 +33,7 @@ def test_master(self):
         scylla_repo = self.url_base + 'unstable/scylla/master/rpm/centos/2021-08-29T00:58:58Z/scylla.repo'
         linux_distro = 'centos'
         version_list = general_test(scylla_repo, linux_distro)
-        self.assertEqual(version_list, ['4.6'])
```

Comment:
Maybe we should ease the test here and check version bigger then 5.0, instead of altering it every release

Reply:
the upgrade test itself has an "auto detect" that could be used here maybe?

Reply:
@fabio, this is the unit test for this "auto detect" stuff, I think :)

Reply:
@fruch, I did it just like we did before: https://github.com/scylladb/scylla-cluster-tests/pull/4179
And I think we can do it again just to not block the PR queue.

@bentsi, wdyt?

Reply:
yes, you aree right... but also, i expect soon master will become 5.1.dev, as 5.0 was already branched

Reply:
those are the numbers coming back from the official repo directory , they master is called nightly there, i.e. 5.1.dev would be under nightly and not on the top level repo list. (like formal release)